tests/pass/intrinsics-x86.rs

```diff
@@ -1,3 +1,51 @@
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
Ah, here's another way to handle tests that should run only on x86. Might be good to also port the other tests (that use the long `ignore-` list) to this pattern.

Reply:
For SSE and SSE2 that will work fine because they are enabled by default. But other features require `-C target-feature=+feature`, which will generate warnings on non-x86.

Maybe a new directive could be added to compiletest? Something like `@only-x86-family`

Reply:
I've done it for SSE and SSE2 for the time being.